test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -13460,6 +13491,12 @@ def __init__(self, username=None, attrs={}, jobname=None):
             self.username = None
         self.du = None
         self.interactive_handle = None
+        if self.platform == 'cray' or self.platform == 'craysim':
+            m = None
+            if 'Resource_List.select' in attrs:
+                m = re.match('.*vntype=.*', attrs['Resource_List.select'])
```

Comment:
Same as above, this is too permissive a re.match().  You can still do the two string matches here as well.